modules/system/assets/ui/vendor/pikaday/css/pikaday.css

```diff
@@ -2,7 +2,7 @@
 
 /*!
  * Pikaday
- * Copyright © 2014 David Bushell | BSD & MIT license | http://dbushell.com/
+ * Copyright © 2014 David Bushell | BSD & MIT license | https://dbushell.com/
```

Comment:
Why were the file permissions changed from 644 to 755 on this?

Reply:
We'll need them changed back to 644 please

Reply:
Sorry I didn't notice the file permissions change. Fixed.